src/core/agent/services/multiSig.types.ts

```diff
@@ -33,6 +34,27 @@ interface MultiSigExnMessage {
         c: any[];
         a: any[];
       };
+      exn: {
```

Comment:
Hmm this isn't right. `InceptMultiSigExnMessage` and `GrantMultiSigExnMessage` have common elements but the embeds are not the same. `icp` only applies to `InceptMultiSigExnMessage` and the embedded grant message (which has ACDC embeds etc) only applies to `GrantMultiSigExnMessage`

Maybe lets call them `InceptMultiSigExn` and `IpexGrantMultiSigExn` too

Reply:
What I'm asking for here is to get the common components.

```
    v: string;
    t: string;
    d: string;
    i: string;
    p: string;
    dt: string;
    r: string;
    q: any;
```

are all common elements for an exn message - find the common elements